--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -223,7 +223,9 @@
     set_target_properties(dumb PROPERTIES INSTALL_NAME_DIR ${CMAKE_INSTALL_FULL_LIBDIR})
 ENDIF(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 
-target_link_libraries(dumb m)
+if(NOT MSVC)
+    target_link_libraries(dumb m)
+endif()
 
 install(FILES ${PKG_CONFIG_FILE} DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 install(FILES ${INSTALL_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
